cmd/wnode-status/config.go

```diff
@@ -69,14 +69,15 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	whisperConfig.TTL = *ttl
 
 	if whisperConfig.EnableMailServer && whisperConfig.PasswordFile == "" {
-		return nil, errors.New("mail server requires -password to be specified")
+		//fixme(@jekamas): hard coded for mail server MVP - issue https://github.com/status-im/status-go/issues/488
+		whisperConfig.Password = "status-offline-inbox"
 	}
 
 	if whisperConfig.EnablePushNotification && whisperConfig.IdentityFile == "" {
 		return nil, errors.New("notification server requires -identity file to be specified")
 	}
 
-	if whisperConfig.PasswordFile != "" {
+	if whisperConfig.PasswordFile != "" && whisperConfig.Password == "" {
```

Comment:
We could add validation that it's not allowed to pass `-passwordfile` and `-password` at the same time and if `-mailserver` is enabled, one of them must be specified.

Reply:
Can you also group dealing with password flags in one place? Now, it's divided by `if whisperConfig.EnablePushNotification && whisperConfig.IdentityFile == "" {`.